dynamic-property-jackson/build.gradle.kts

```diff
@@ -5,12 +5,12 @@ plugins {
 
 dependencies {
 
-    compile(project(":dynamic-property-api"))
+    implementation(project(":dynamic-property-api"))
```

Comment:
According to [this section](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_recognizing_dependencies), `api`'s use cases include:

>types used in super classes or interfaces

JacksonDynamicPropertyMarshaller is a subtype of DynamicPropertyMarshaller, which corresponds to `dynamic-property-api` module

It seems, that `project(":dynamic-property-api")` must be an `api` dependency

Reply:
you're right, my fault. I re-read the docs, now fixed it